boards/nxp/lpcxpresso55s36/lpcxpresso55s36.yaml

```diff
@@ -20,3 +20,5 @@ supported:
   - pwm
   - dac
 vendor: nxp
+env: # Provisional hack to prevent this board from running, as it fails in many test & samples
+  - LPCXPRESSO55S36_TWISTER_ENABLE
```

Comment:
is this documented anywhere? I'm sure it works but I can't figure how and how you came up with it :-)

Reply:
You mean the environment variable filter? or the funky name of the environment variable I came up with? :)
If you mean the first, I just checked, and you are right this is not documented in https://docs.zephyrproject.org/latest/develop/test/twister.html#board-configuration 
So either one happens to already know it, or by finding it in the code:
https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/pylib/twister/twisterlib/platform.py#L106-L110
=> we should add it in the doc

Reply:
oh it's just an env that does not match anything so the board does not run, got it, in fairness you did write it was a hack, what did I expect :-)